python/hail/docs/conf.py

```diff
@@ -45,6 +49,30 @@
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['_templates', '_templates/_autosummary']
 
+doctest_global_setup = """import os
+from hail import *
+
+if not os.path.isdir("output/"):
+    os.mkdir("output/")
+
+vds_files = ["sample.vds", "sample.qc.vds", "sample.filtered.vds"]
+for vds in vds_files:
+    if os.path.isdir(vds):
+        shutil.rmtree(vds)
+
+hc = HailContext(log="output/hail.log", quiet=True)"""
+
+doctest_global_cleanup = """import shutil, os
+
+hc.stop()
+
+shutil.rmtree("output/")
```

Comment:
would prefer cleanup at beginning, so failed tests don't cause subsequent builds to fail.  Or both